elife/config/etc-init-goaws-init.conf

```diff
@@ -0,0 +1,11 @@
+description "GOAWS will run a local AWS server that can be used by setting --endpoint=http://localhost:4100 to commands"
+respawn
+respawn limit 10 30
+kill timeout 70
+setuid {{ pillar.elife.deploy_user.username }}
```

Comment:
When I SSH into the box and run it manually it works, so I imagine that is the elife user